website/source/intro/getting-started/install.html.md

```diff
@@ -38,6 +47,29 @@ for instructions on setting the PATH on Linux and Mac. [This
 page](https://stackoverflow.com/questions/1618280/where-can-i-set-path-to-make-exe-on-windows)
 contains instructions for setting the PATH on Windows.
 
+## Compiling from Source
+
+To compile from source, you will need [Go](https://golang.org) installed and
+configured properly (including a `GOPATH` environment variable set), as well
```

Comment:
This is a bit dated, Go removed the requirement to have `GOPATH` defined and defaults to `$HOME/go`.